packages/grpc-native-core/src/server.js

```diff
@@ -567,14 +576,15 @@ ServerDuplexStream.prototype.waitForCancel = waitForCancel;
  * @param {grpc.Metadata} metadata Metadata from the client
  */
 function handleUnary(call, handler, metadata) {
+  var asyncResource = createAsyncResourceWrapper('GrpcServerRequest');
```

Comment:
As suggested by the [official docs](https://nodejs.org/dist/latest-v9.x/docs/api/async_hooks.html#async_hooks_type), it would be best to use the module name as a prefix. Something like 'grpc.ServerRequest'.

Reply:
Done